Do not ignore `src/` when publishing NPM

huan opened the pull request:
See: https://github.com/tensorflow/tfjs/issues/1339

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/tensorflow/tfjs-core/1611)
<!-- Reviewable:end -->

dsmilkov commented:
I'd like to better understand the usability before we make this decision. See https://github.com/tensorflow/tfjs/issues/1339#issuecomment-471193352 for an alternative and let me know if that works for you. Thanks!

huan commented:
HI @dsmilkov , thank you for the reply and not closing this PR.

As I replied in the other issue, I will lose the interest to debug the tfjs library if I can not found the source code when I'm working in the terminal.

huan commented:
For example: how could I get to know what's wrong from my following error trace log?

```
# npm i @tensorflow/tfjs-node@next

> @tensorflow/tfjs-node@1.0.0-alpha2 install /notebooks/node_modules/@tensorflow/tfjs-node
> node scripts/install.js

* Downloading libtensorflow
[==============================] 2922683/bps 100% 0.0s
* Building TensorFlow Node.js bindings
+ @tensorflow/tfjs-node@1.0.0-alpha2
removed 21 packages, updated 1 package and audited 140 packages in 20.35s
found 0 vulnerabilities

root@94a0ba4be94e:/notebooks# node -r ts-node/register translation.ts dist/fra.txt 
2019-03-09 16:56:43.196524: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
Number of samples: 10000
Number of unique input tokens: 69
Number of unique output tokens: 93
Max sequence length for inputs: 16
Max sequence length for outputs: 59
Saved metadata at:  /tmp/translation.keras/metadata.json
Orthogonal initializer is being called on a matrix with more than 2000 (262144) elements: Slowness may result.
Orthogonal initializer is being called on a matrix with more than 2000 (262144) elements: Slowness may result.
Epoch 1 / 20
(node:3814) UnhandledPromiseRejectionWarning: Error: Not yet implemented
    at NodeJSKernelBackend.KernelBackend.zerosLike (/notebooks/node_modules/@tensorflow/tfjs-core/src/kernels/backend.ts:597:11)
    at environment_1.ENV.engine.runKernel.$x (/notebooks/node_modules/@tensorflow/tfjs-core/src/ops/tensor_ops.ts:467:50)
    at /notebooks/node_modules/@tensorflow/tfjs-core/src/engine.ts:206:22
    at Engine.scopedRun (/notebooks/node_modules/@tensorflow/tfjs-core/src/engine.ts:167:19)
    at Engine.runKernel (/notebooks/node_modules/@tensorflow/tfjs-core/src/engine.ts:202:10)
    at zerosLike_ (/notebooks/node_modules/@tensorflow/tfjs-core/src/ops/tensor_ops.ts:467:21)
    at Object.zerosLike (/notebooks/node_modules/@tensorflow/tfjs-core/src/ops/operation.ts:46:24)
    at Object.$x (/notebooks/node_modules/@tensorflow/tfjs-core/src/ops/unary_ops.ts:349:21)
    at _loop_1 (/notebooks/node_modules/@tensorflow/tfjs-core/src/tape.ts:171:43)
    at Object.backpropagateGradients (/notebooks/node_modules/@tensorflow/tfjs-core/dist/tape.js:158:9)
(node:3814) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:3814) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
root@94a0ba4be94e:/notebooks# 
```
